lib/compilers/rustc-cg-gcc.js

```diff
@@ -32,6 +32,7 @@ export class RustcCgGCCCompiler extends RustCompiler {
     constructor(info, env) {
         super(info, env);
         this.compiler.supportsIrView = false;
+        this.compiler.supportsRustMirView = false;
```

Comment:
Does this work? I don't have rustc-cg-gcc locally so I was unable to test.

Reply:
@dkm may know about this.

Reply:
I should try it, but I don't see why it would not work, this "compiler" is only changing the backend, so MIR output should work correctly. I'll try this now and let you know!

Reply:
As expected, it works fine, so you can revert this change :)
![Screenshot_2021-07-21 Compiler Explorer](https://user-images.githubusercontent.com/87603/126496708-99771680-b26c-40a5-85ce-e679c5471c25.png)

Reply:
Okay, re-enabled it. Thank you for checking!